config.w32

```diff
@@ -45,6 +45,7 @@ function mongodb_parse_version_file(inpath, prefix)
 
 ARG_ENABLE("mongodb", "MongoDB support", "no");
 ARG_WITH("mongodb-sasl", "MongoDB: Build against Cyrus-SASL", "yes");
+ARG_ENABLE("client-side-encryption", "MongoDB: Enable client-side encryption", "auto");
```

Comment:
From my understanding of [`ARG_ENABLE`](https://github.com/php/php-src/blob/5d6e923d46a89fe9cd8fb6c3a6da675aa67197b4/win32/build/confutils.js#L232), the default value is an arbitrary string and there's no enforcement of possible values. I've noted that within this file, we only care that the value is not `"no"`.